src/device_hub.cpp

```diff
@@ -18,6 +18,15 @@ namespace librealsense
         {
             bool filtered = false;
             auto data = dev->get_device_data();
+            for (const auto& usb : data.usb_devices)
```

Comment:
So TM2 was just skipped... It is weird that the usb screening was not in place and yet it worked